java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java

```diff
@@ -35,6 +37,7 @@
 import org.apache.commons.fileupload.servlet.ServletFileUpload;
 import org.apache.commons.fileupload.util.Streams;
 import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.StringEscapeUtils;
```

Comment:
For later, also consider:
https://github.com/google/guava/blob/master/guava/src/com/google/common/html/HtmlEscapers.java
The Apache one does entity escaping, which in theory can be trouble (however in this case no expected values are going to cause issues, so that shouldn't be an issue).